src/lights/PointLightShadow.js

```diff
@@ -58,7 +58,7 @@ PointLightShadow.prototype = Object.assign( Object.create( LightShadow.prototype
 
 	isPointLightShadow: true,
 
-	updateMatrices: function ( light, viewCamera, viewportIndex ) {
+	updateMatrices: function ( light, viewportIndex = 0 ) {
```

Comment:
Please avoid the usage of default parameters. Write it like so:
```js
viewportIndex = viewportIndex || 0;
```

Reply:
@mrdoob told me last time I used `|| 0` to be more explicit, so I'll follow that here if you don't mind.